
package Protos;


message TimeSeries {
  	repeated double data = 1 [packed=true];
}

message LabeledTimeSeries {
    required string label = 2;
	required TimeSeries ts = 3;
}

message LabeledTimeSeriesList {
    repeated LabeledTimeSeries list = 1;
}

message PatternsTimeline {
    repeated string labels = 1;
    repeated uint32 labels_id_timeline = 2;
    repeated double timeline = 3;
    required double tmax = 4;
    required double dt = 5;
    required double gap_between_patterns = 6;
}


